androidshared/src/main/java/org/odk/collect/androidshared/livedata/LiveDataUtils.java

```diff
@@ -11,6 +16,20 @@ private LiveDataUtils() {
 
     }
 
+    public static <T> Cancellable observe(LiveData<T> liveData, Consumer<T> consumer) {
```

Comment:
This was just to clean up some of the boiler plate around using `LiveData`'s `observerForever` with our existing `Cancellable` abstraction.